go/libkb/id_table.go

```diff
@@ -990,10 +990,23 @@ func (s *WalletStellarChainLink) VerifyReverseSig(_ ComputedKeyFamily) (err erro
 }
 
 func (s *WalletStellarChainLink) Display(m MetaContext, ui IdentifyUI) error {
+	// First get an up to date user card, since hiding the Stellar address affects it.
+	card, err := UserCard(m, s.GetUID(), true)
```

Comment:
i think this breaks doing `keybase id` while logged out.
`true` is `useSession`